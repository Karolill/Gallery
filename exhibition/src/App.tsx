import React from "react";
import Carousel from "./components/Carousel/carousel";

function App() {
  return (
    <div className="App">
      <Carousel />
    </div>
  );
}

export default App;
