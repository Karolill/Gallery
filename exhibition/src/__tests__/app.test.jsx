import React from "react";
import { render, cleanup } from "@testing-library/react";
import App from "../App";

afterEach(cleanup);

/*To  test that the component hierarchy is the way we expect it to be*/

it("installationInfo snapshot", () => {
  const { container } = render(<App />);
  expect(container).toMatchSnapshot();
});
