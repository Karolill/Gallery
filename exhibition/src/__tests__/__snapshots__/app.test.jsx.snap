// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`installationInfo snappshot 1`] = `
<div>
  <div
    class="App"
  >
    <header>
      <h1
        id="title"
      >
        Geometry gallery
      </h1>
    </header>
    <div>
      <div
        class="closed"
        id="menu"
      >
        <div
          class="closed"
          id="close-button"
        >
          <div
            class="line-one"
          >
            <div
              class="line-two"
            />
          </div>
        </div>
        <button
          id="displayFavButton"
          type="button"
        >
          Only display favorites
        </button>
        <div
          class="closed"
          id="up-arrow"
        />
      </div>
      <div
        class="closed"
        id="burger-button"
      >
        <div
          class="closed"
          id="top-line"
        />
        <div
          class="closed"
          id="middle-line"
        />
        <div
          class="closed"
          id="bottom-line"
        />
      </div>
      <div
        class="closed"
        id="blurry-area"
      />
    </div>
    <div
      id="carousel"
    >
      <div
        class="arrows left"
      >
        <div
          class="leftArrow"
        />
      </div>
      <div
        id="installation"
      >
        <div>
          <svg
            class="picture"
            height="350"
            width="350"
          >
            <rect
              fill="#4e4e4e"
              height="350"
              id="i1_r1"
              stroke="#4e4e4e"
              width="350"
              x="0"
              y="0"
            />
            <circle
              cx="156"
              cy="78"
              fill="black"
              id="i1_c1"
              r="30"
              stroke="black"
              stroke-width="0"
            />
            <circle
              cx="130"
              cy="276"
              fill="black"
              id="i1_c2"
              r="60"
              stroke="black"
              stroke-width="0"
            />
            <circle
              cx="300"
              cy="175"
              fill="black"
              id="i1_c3"
              r="35"
              stroke="black"
              stroke-width="0"
            />
            <circle
              cx="275"
              cy="40"
              fill="black"
              id="i1_c4"
              r="20"
              stroke="black"
              stroke-width="0"
            />
            <circle
              cx="60"
              cy="200"
              fill="black"
              id="i1_c5"
              r="30"
              stroke="black"
              stroke-width="0"
            />
            <circle
              cx="25"
              cy="30"
              fill="black"
              id="i1_c6"
              r="40"
              stroke="black"
              stroke-width="0"
            />
            <circle
              cx="270"
              cy="350"
              fill="black"
              id="i1_c7"
              r="55"
              stroke="black"
              stroke-width="0"
            />
          </svg>
        </div>
        <div
          class="poem"
          id="poem1"
        />
      </div>
      <div
        class="audio-control"
      >
        <audio
          controls=""
        >
          <source
            src="https://docs.google.com/uc?export=download&id=1gAzs4UwdenQmNbEm9mdDIunb8Ef-kn5i"
          />
          Your browser does not support audio-tag
        </audio>
      </div>
      <div
        class="arrows right"
      >
        <div
          class="rightArrow"
        />
      </div>
      <div
        id="artInfo"
      >
        <div>
          <h2
            id="installation-title"
          >
            Dark dots
             (
            2000
            )
          </h2>
          <h4
            id="artist"
          >
            Mary Cassatt
          </h4>
        </div>
      </div>
      <button>
        Add to favorites
      </button>
    </div>
  </div>
</div>
`;
